ktor-server/ktor-server-core/common/src/io/ktor/server/request/ApplicationReceiveFunctions.kt

```diff
@@ -188,9 +188,11 @@ public var ApplicationCall.formFieldLimit: Long
  * @throws ContentTransformationException when content cannot be transformed to the [MultiPartData].
  */
 public suspend inline fun ApplicationCall.receiveMultipart(
-    formFieldLimit: Long = DEFAULT_FORM_FIELD_MAX_SIZE
```

Comment:
it looks like we need old method with hidden for backward compatibility

Reply:
Isn't it the same bytecode to change an arg to nullable?  I think don't think it could break anything...

Reply:
it changes L to Long, so yep, it's different

Reply:
We could use `-1` instead of `null` here to keep this function backward compatible

Reply:
Oh yeah that could work 👍